package.json

```diff
@@ -16,9 +16,10 @@
     "node": "6.x"
   },
   "dependencies": {
-    "bootstrap-sass": "^3.3.7",
+    "bootstrap": "^4.0.0-alpha.5",
     "classnames": "^2.2.5",
     "dropzone": "^4.2.0",
+    "font-awesome": "^4.7.0",
```

Comment:
My comment still holds: please include the icon as SVG instead of adding an entire font family to our requirements. We should drop all icon fonts.